api/Element.json

```
@@ -6511,7 +6511,7 @@
             "edge": [
               {
                 "version_added": "18",
-                "notes": "No support for <code>smooth</code> behavior."
+                "notes": "Parameter <code>alignToTop</code> is supported. No support for <code>smooth</code> behavior."
```

If I understand the data correctly, the note about the smooth option is redundant with the `scrollIntoViewOptions` feature data you updated below. In that case, I'd suggest rephrasing this

```suggestion
                "notes": "The only parameter supported is <code>alignToTop</code>."
```